### ISSUE TITLE
Improvement to dark mode selection

### ISSUE THREAD

**Issue by andybee** [architect/arc.codes#522]
As I understand the current dark mode implementation looks at the current browser/OS setting first and sets that as a default value, before letting the user customise via the top right toggle.

I have my OS set to flip from light to dark based on sunset/sunrise, so if I visit the site for the first time during the day I'm on the light theme and have to toggle it at sunset to dark, then back to light again the next day etc. (curse you, short daylight of winter!)

Could we add a third option for "OS/Browser default", or not persist a default setting and always check the browser's opinion on page load?

**Comment by filmaj** [comment]
Is doing that even possible? Not sure if I've seen that in the wild.

**Comment by andybee** [comment]
Yep, it's a bit of jiggery pokery with either setting an explicit class on the body/html tag to force dark/light or relying on `prefer-colour-scheme` media selector.

https://web.dev/articles/prefers-color-scheme#the_prefers-color-scheme_media_query
